Calculator/Parser/ParseError/ParseError.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ParseError.swift
+//  Calculator
+//
+//  Created by Gleb Kovalenko on 17.07.2022.
+//
+
+import Foundation
+
+// MARK - ParseError
+
+enum ParseError: Error {
+    case oneMoreDot
+    case unknownFunction
+    case unknownOperation
```

Comment:
Показывай, что конкретно неправильно

```suggestion
    case unknownFunction(String)
    case unknownOperation(String)
```